src/cache.jl

```diff
@@ -94,7 +94,11 @@ end
 updatecache(absentmodule::Symbol, server) = updatecache([absentmodule], server)
 
 function updatecache(absentmodules::Vector{Symbol}, server)
+    server.cp_lock && return
```

Comment:
In practice 'give up' means: next time that a key is pressed, the mouse hovers etc. then we'll see if we can launch a new child process. 

I've found package loading to be an action ripe for segfault-y crashing (build errors etc.) and a julia process can really take up a bit too much RAM which is why I don't particularly fancy a persistent child process.

In our super new Parser I've got a workspace/directory parsing system that tracks all module use in a more centralised manner which also allows us to track packages that refuse to load.